Cargo.toml

```diff
@@ -12,11 +12,15 @@ the rest of the tokio crates.
 """
 
 [dependencies]
-futures = "0.1.4"
+futures = "0.1"
 log = "0.3"
 mio = "0.6"
 scoped-tls = "0.1.0"
 slab = "0.3"
 
 [dev-dependencies]
 env_logger = { version = "0.3", default-features = false }
+nix = "0.7"
```

Comment:
I managed to not find `io::Error::last_os_error()`, so these "arguments" of mine mostly fall flat, as I see from your cleanup patch.